guide/guide/beginner/basicbot.md

```diff
@@ -95,32 +95,34 @@ const Discord = require("discord.js");
 const client = new Discord.Client();
 
 client.on("ready", () => {
-    const GCommandsClient = new GCommands(client, {
-        cmdDir: "commands/",
-        eventDir: "events/",
-        unkownCommandMessage: false, // true of false | send unkownCommand Message
-        language: "english", //english, spanish, portuguese, russian, german, czech, turkish
-        slash: {
-           slash: 'both', //true = slash only, false = only normal, both = slash and normal
-           prefix: '.' // for normal commands
-        },
-        defaultCooldown: 3,
-        database: "url"
-        /* DB SUPPORT
-         * redis://user:pass@localhost:6379
-         * mongodb://user:pass@localhost:27017/dbname
-         * sqlite://path/to/database.sqlite
-         * postgresql://user:pass@localhost:5432/dbname
-         * mysql://user:pass@localhost:3306/dbname
-        */
-    })
-    
-    GCommandsClient.on("log", (log) => {console.log(log)})
-
-    console.log("Ready")
-})
-
-client.login("Bot token here")
+  const GCommandsClient = new GCommands(client, {
+    cmdDir: "commands/",
+    eventDir: "events/",
+    unkownCommandMessage: false, // true of false | send unkownCommand Message
+    language: "english", //english, spanish, portuguese, russian, german, czech, turkish
+    slash: {
+      slash: "both", //true = slash only, false = only normal, both = slash and normal
+      prefix: ".", // for normal commands
+    },
+    defaultCooldown: 3,
+    database: "url",
+    /* DB SUPPORT
+     * redis://user:pass@localhost:6379
+     * mongodb://user:pass@localhost:27017/dbname
+     * sqlite://path/to/database.sqlite
+     * postgresql://user:pass@localhost:5432/dbname
+     * mysql://user:pass@localhost:3306/dbname
+     */
+  });
+
+  GCommandsClient.on("log", (log) => {
+    console.log(log);
+  });
+
+  console.log("Ready");
+});
+
+client.login("Bot token here");

```

Comment:
```suggestion
const { GCommandsClient } = require("gcommands");

const client = new GCommandsClient({
  intents: ["Your intents"]
  cmdDir: "commands/",
  eventDir: "events/",
  unkownCommandMessage: false, // true of false | send unkownCommand Message
  language: "english", //english, spanish, portuguese, russian, german, czech, turkish
  slash: {
    slash: "both", //true = slash only, false = only normal, both = slash and normal
    prefix: ".", // for normal commands
  },
  defaultCooldown: 3,
  database: "url",
  /* DB SUPPORT
   * redis://user:pass@localhost:6379
   * mongodb://user:pass@localhost:27017/dbname
   * sqlite://path/to/database.sqlite
   * postgresql://user:pass@localhost:5432/dbname
   * mysql://user:pass@localhost:3306/dbname
   */
});

client
   .on("ready", console.log("ready"))
   .on("log", console.log)

client.login("Bot token here");
```

Put in a guide that in `GCommandsClient` can use gcommands options but also discordjs client options for defined intents and stuff like that.